src/lint/track_config.nim

```diff
@@ -237,7 +248,7 @@ iterator visibleConceptExercises(trackConfig: TrackConfig): ConceptExercise =
   ## Yields every concept exercise in `trackConfig` that has a `status` of
   ## "beta" or "active".
   for conceptExercise in trackConfig.exercises.`concept`:
-    if conceptExercise.status in [sBeta, sActive]:
+    if conceptExercise.status in [sMissing, sBeta, sActive]:
```

Comment:
Should the above comment be updated to accommodate the newly added status?

Reply:
Yes, thanks. I'll amend #414.